tv_apps/smart-home/js/home.js

```diff
@@ -296,7 +342,7 @@
     },
 
     onMove: function(key) {
-      if (this.edit.onMove(key)) {
+      if (this.edit.onMove(key) || !this.isNavigable) {
```

Comment:
Should we check isNavigable first than this.edit.onMove(key)??
We can move the elements while animation is running.